bxml/bxmlOverview.md

```diff
@@ -0,0 +1,61 @@
+
+## Assumptions
+
+* You have a way to recieve way to recieve webhooks.
+* You have downloaded [Postman](https://www.getpostman.com) or have another way to make Post requests to the API.
+* You have your [Voice and Messaging API](https://app.bandwidth.com) {userId}, {apiToken}, {apiSecret}
+* You have a [Bandwidth phone number](https://dev.bandwidth.com/howto/phonenumbers.html).
+
+## About BXML
+
+BXML is completely separate from the REST API. When using the BXML, the REST API can only be used to make calls. Here is an example of a common call flow:
+![BXML Flow](BXMLCallFlow.png)
+
+## Setup
+
+For a step by step guide on downloading the software and adding your Bandwidth credentials, visit the [Get Started](http://dev.bandwidth.com/get-started.html)
+
+## Create Call
+
+To create the call, we will use Bandwidth’s REST API. **THIS IS THE ONLY TIME WE WILL USE THE REST API IN THIS APPLICATION**. Create a call with the `to`, `from`, `callbackUrl`, and `callbackHttpMethod` parameters. The `callbackHttpMethod` needs to be set to <code class="get">GET</code> for BXML. The `callbackUrl` is also important because this is how Bandwidth will notify your program that the call has started and your program can then send the BXML information to the call. For a step by step guide on creating the call, reference the [Get Started Guide](http://dev.bandwidth.com/get-started.html) or for a quick overview, visit the [Outbound Call Building Block](http://dev.bandwidth.com/howto/outboundCall.html).
+
+```http
+POST https://api.catapult.inetwork.com/v1/users/{{userId}}/calls HTTP/1.1
+Content-Type: application/json; charset=utf-8
+Authorization: {apiToken:apiSecret}
+
+{
+	"to"                 : "+12345678902",
+	"from"               : "+12345678901",
+	"callbackUrl"        : "{callbackUrl}",
+	"callbackHttpMethod" : "GET"
+}
+```
+
+## Handle Incoming Calls
+**Make sure auto answer is on**
+### Using the Voice and Messaging Dashboard
+
+To allow the program to answer incoming calls, login to the [app.bandwidth.com](https://app.bandwidth.com) dashboard. Either create a new [application](https://app.bandwidth.com/applications/manage/) or edit an existing [application](https://app.bandwidth.com/applications/manage/). The callback request method should be <code class="get">GET</code> and the application type should be set to Voice. Add a callback URL and toggle “Automatically answer incoming calls” to on. Finally, add a Bandwidth phone number to this application. This number will be the number users can call. When the user calls the bandwidth number, the number will answer and notify the callback url that there is an active call. From there, the program acts the same way as create call.
+![BXML Application Setup](bxmlApplicationSetup.png)
```

Comment:
Picture not showing up

![image](https://user-images.githubusercontent.com/6805534/36688716-451a3782-1afb-11e8-99ef-664db1fa8820.png)

Reply:
How should I fix it? It is visible when I click the preview changes link

Reply:
Doesn't look like it picture was added to git. Try checking `git status` on your local machine to to see if it's there just not added to the repo.  Also try adding a new line between the end of the paragraph and the `![BXML Application Setup](bxmlApplicationSetup.png)`